pkg/controller/elasticsearch/client/mock.go

```diff
@@ -33,6 +34,18 @@ func NewMockClientWithUser(v version.Version, u BasicAuth, fn RoundTripFunc) Cli
 	return versioned(baseClient, v)
 }
 
+func NewFullMockClient(v version.Version, u BasicAuth, fn RoundTripFunc, endpoint string, caCerts []*x509.Certificate) Client {
```

Comment:
Not sure if we need this. Could you not just use the regular client constructor function `esclient.NewElasticsearchClient` for the test where this is used?

Reply:
Yes, we can. Done.